pkg/proxy/util/nodeport_addresses_test.go

```diff
@@ -379,7 +379,7 @@ func TestGetNodeIPs(t *testing.T) {
 			}
 
 			for _, family := range []v1.IPFamily{v1.IPv4Protocol, v1.IPv6Protocol} {
-				npa := NewNodePortAddresses(family, tc.cidrs)
+				npa := NewNodePortAddresses(family, tc.cidrs, nil)
```

Comment:
should add a unit test for the new behavior as well

Reply:
I've addressed all your comments. PTAL if you have time. Thanks for reviewing.